source/uwp/Renderer/lib/AdaptiveChoiceSetInputRenderer.cpp

```diff
@@ -169,6 +169,28 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
                     // Otherwise, leave all options unset
                     radioButton.IsChecked(IsChoiceSelected(values, input));
                 }
+
+                radioButton.PreviewKeyDown(
+                    [this, stackPanel](winrt::IInspectable const& sender, winrt::KeyRoutedEventArgs const& args) -> void
+                    {
+                        std::uint32_t currentIndex;
+                        auto children = stackPanel.Children();
+                        auto size = children.Size();
+                        if (const auto isButtonFound = children.IndexOf(sender.as<winrt::RadioButton>(), currentIndex))
+                        {
+                            if (args.Key() == winrt::VirtualKey::Down)
+                            {
+                                currentIndex = (currentIndex + 1) % size;
```

Comment:
It might be a good idea to create a new variable `nextButtonIndex` or something like that to hold this number for better readability :)